packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -207,7 +207,8 @@ class PointerEventConverter {
               kind: kind,
               device: datum.device,
               position: position,
-              buttons: datum.buttons,
+              // TODO(tongmu): Move button patching to embedder, https://github.com/flutter/flutter/issues/30454
+              buttons: datum.buttons | kPrimaryButton,
```

Comment:
as written won't that break mice?

i would expect this to only happen for kind = touch and kind = stylus devices while down.

Reply:
You're right, that's a stupid mistake. I'll fix it ASAP.